src/input_reader.rs

```diff
@@ -258,9 +265,30 @@ impl Default for ActionSources {
     }
 }
 
-/// Tracks all consumed input from Bevy resources.
+/// All consumed input by actions in each schedule.
```

Comment:
Brief context of why this exists would be really helpful to record here.

Reply:
This and the ones below are private structs, but I agree we should document them since the logic is quite tricky.